### ISSUE TITLE
App crashes when trying to add to many values

### ISSUE THREAD

**Issue by 0xLeif** [0xLeif/FLite#15]
For this test
```
func testTodoArray() {
        let semaphore = DispatchSemaphore(value: 0)
        var values = (0 ..< 500).map { _ in Todo(title: "Todo #\(Int.random(in: 0 ... 10000))", strings: []) }
        
        FLite.storage = .memory
        
        FLite.manager.set(maxConnections: 10)
        
        FLite.manager.set(id: "Something Else")
        
        FLite.prepare(model: Todo.self) {
            "Prepared".log()
        }
        
        values.forEach { value in 
            FLite.create(model: value) {
                "Created: \($0)".log()
            }
        }
        
        FLite.fetch(model: Todo.self) { qb in
            qb.all()
                .whenSuccess {
                    "Value: \($0)".log()
                    values = $0
                    semaphore.signal()
            }
        }
        
        semaphore.wait()
        XCTAssert(values.count > 0)
    }
```

Trying to add 500 values
```
var values = (0 ..< 500).map { _ in Todo(title: "Todo #\(Int.random(in: 0 ... 10000))", strings: []) }
```

with 10 maxConnections
```
FLite.manager.set(maxConnections: 10)
```

Error: `NIO-ELT-#0 (9): EXC_BAD_ACCESS (code=2, address=0x700001ebeff8)`
```
warning: could not execute support code to read Objective-C class data in the process. This may reduce the quality of type information available.
```

**Comment by 0xLeif** [comment]
Backtrace
[bt.txt](https://github.com/0xLeif/FLite/files/4628988/bt.txt)

**Comment by 0xLeif** [comment]
Fixed in #13
